lib/chore/worker.rb

```diff
@@ -86,7 +86,7 @@ def start_item(item)
 
       begin
         Chore.logger.info { "Running job #{klass} with params #{message}"}
-        perform_job(klass,message)
+        perform_job(klass, item.id, message, item.queue_timeout)
```

Comment:
I tried to pass just `item` here, but for reasons I TRULY DO NOT UNDERSTAND, whatever the first parameter passed into `perform_job` is `includes` the `job` class, which triggers https://github.com/Tapjoy/chore/pull/50/files#diff-41d7585164fb76e745b1878b6650d4ebR26, so the first parameter _has_ to be the `klass`. In for a penny, in for a pound, so I just decided to make the other parameters explicit as well.

If anyone can understand why this is happening I'd be very interested.